src/wp-includes/script-loader.php

```diff
@@ -34,6 +34,10 @@
 /** WordPress Styles Functions */
 require ABSPATH . WPINC . '/functions.wp-styles.php';
 
+if ( ! defined( 'WP_ASSET_SUFFIX_MIN' ) ) {
+	define( 'WP_ASSET_SUFFIX_MIN', '.min' );
+}
```

Comment:
This should probably be moved to the `wp-includes/default-constants.php` file.

Reply:
Yes, I wasn't sure where to put that.

Reply:
Well, now you know 😁

Reply:
Relocated in commit https://github.com/WordPress/wordpress-develop/pull/3340/commits/caab8d53e0ceaa0f980a2ed069541efa5131facf.